samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/Widgets.razor

```diff
@@ -49,18 +49,24 @@
 <button @onclick="@(() => MapView!.Refresh())">Refresh</button>
 
 <MapView @ref="MapView" Longitude="-40" Latitude="28" Zoom="2" Class="map-view">
-    <Map ArcGISDefaultBasemap="arcgis-navigation" />
+    <Map>
+        <Basemap>
```

Comment:
any particular reason for the change here? `ArcGISDefaultBasemap` is correct?  Is this to establish a element build pattern `Map` -> `Basemap` -> `BasemapStyle` vs using just `Map`?